community/contributing/content_guidelines.rst

```diff
@@ -0,0 +1,95 @@
+.. _doc_content_guidelines:
+
+Content guidelines
+==================
+
+This document is here to help us assess what we should include in the official
+documentation. Below, you will find a couple of principles and recommendations
+to write accessible content.
+
+We want to achieve two goals:
+
+1. **Empathize with our users**. We should write in a way that makes it easy for
+   them to learn from the docs.
+2. **Write a complete reference manual**. Our goal here is not to teach
+   programming foundations. Instead, we should provide a reference for how
+   Godot’s features work.
+
+Guidelines and principles
+-------------------------
+
+Below are the guidelines we should strive to follow. They are not hard rules,
+though: exceptionally, a topic will require breaking one or more of these.
+Still, we should strive to achieve the two goals listed above.
+
+Writing complete and accessible documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**A feature doesn’t exist unless it is documented**. If a user can’t find
+information about a feature and how it works, it doesn’t exist to them. We
+should ensure that we cover everything Godot does.
+
+.. note::
+
+   When adding or updating an engine feature, the documentation team needs to
+   know about it. Contributors should open an issue on the godot-docs repository
+   when their work gets merged and requires documentation.
+
+Do your best to keep documents **under 1000 words in length**. If a page goes
+past that threshold, consider splitting it into two parts, if possible. Limiting
+page size forces us to write concisely and to break large documents so they each
+focus on a particular problem.
+
+Make it clear what **problem** each page or section of a page tackles and what
+the user will learn from it. Users need to know if they’re reading the correct
+guide to solving problems they encounter. For example, instead of writing the
+heading “Signals”, consider writing “Reacting to changes in another object with
+signals”. While a bit long, the second title makes it clear what is the purpose
+of signals.
+
+If the page assumes specific knowledge of other Godot features, mention it and
+link it to the corresponding documentation. For instance, a page about physics
+may use signals, in which case we could note that the page that introduces
+signals is a pre-requisite.
+
+Limiting cognitive load
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Limit the cognitive load required to read the documentation. The simpler and
+more explicit language we use, the more efficient it becomes for people to
+learn. You can do so by:
+
+1. Introducing only one new concept at a time whenever possible.
+2. Using simple English, as we recommend in our writing guidelines.
+3. Include one or more **concrete usage examples**. Prefer a real-world example
+   to abstract code like ``foobar``.
```

Comment:
```suggestion
3. Include one or more **concrete usage examples**. It is preferable to use a 
real-world example as opposed to abstract code concepts like ``foobar``.
```

Reply:
We tend not to use the passive voice, as per [Docs writing guidelines](https://docs.godotengine.org/en/stable/community/contributing/docs_writing_guidelines.html).

Reply:
I'm not very good at avoiding passive voice, however I still wouldn't start the sentence with Prefer. I'd suggest perhaps "Use a real-world example over abstract code like ``foobar``" instead.

Reply:
In this case, the passive sounds fine, it's a bit softer than using the imperative form.